test/remote_docker.py

```diff
@@ -0,0 +1,54 @@
+import os
+import argparse
+from subprocess import check_output, check_call
+import socket
+import random
+
+def detect_port(port):
+    '''Detect if the port is used, return True if the port is used'''
+    socket_test = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        socket_test.connect(('127.0.0.1', int(port)))
+        socket_test.close()
+        return True
+    except:
+        return False
+
+def find_port():
+    '''Find a port which is free'''
+    port = random.randint(10000, 20000)
+    while detect_port(port):
+        port = random.randint(10000, 20000)
+    return port
+
+def start_container(image, name):
+    '''Start docker container'''
```

Comment:
Please comment where the port file is stored.

Reply:
added comment.